README.md

```diff
@@ -44,10 +60,26 @@ For basic setuptools installations:
 $ pip install setuptools
 
 # Install the package
-$ cd rd-apmm-python-oauth
+$ cd nmos-auth-server
 $ sudo python setup.py install
 ```
 
+### Testing
+
+The test tooling requires the tox package. To install run:
+
+```bash
+pip install tox
+```
+
+The unit tests can then be run with:
+
+```bash
+make test
+```
+
+from the root directory of the repository.
```

Comment:
Not a fan of encouraging makefile use just for cross compatability reasons (and also no one likes them.)

```suggestion
Testing of the package can be achieved using tox:
```bash
# Install tox
pip install tox

# Run tests using tox environment for Python3
cd nmos-auth-server
tox -e py3
```

```

Reply:
Github has removed the trailing ``` that needs to be on the end of that suggestion